### ISSUE TITLE
代码问题

### ISSUE THREAD

**Issue by fan609** [VisionXLab/point2rbox-v2#11]
大佬你好，Point2RBoxV2Head.py文件的_get_targets_single函数中是不是使用真实标签的长宽计算面积，为每个先验点分配gt点，这个应该是利用了长宽先验信息吧？且center_r也用到长宽的信息计算，这个是不是算是利用真实标签的长宽的先验信息进行标签分配不是严格的点监督？
“ 
areas = gt_bboxes.areas
..................
center_r = torch.clamp((w * h).sqrt() / 64, 1, 5)[..., None]
..................
 inside_gt_bbox_mask = (abs(offset) < stride * center_r).all(dim=-1)
..................
areas[inside_gt_bbox_mask == 0] = INF
areas[inside_regress_range == 0] = INF
min_area, min_area_inds = areas.min(dim=1)

”

**Comment by yuyi1005** [comment]
The gt widths/heights are predicted by Point2RBox-v2 itself, in [point2rbox_v2.py](https://github.com/VisionXLab/point2rbox-v2/blob/d8869d3444e45d35929ff8e215af35b5d23022dd/mmrotate/models/detectors/point2rbox_v2.py#L332).
